packages/fuels-test-helpers/src/chains.rs

```diff
@@ -53,35 +50,50 @@ async fn no_op_signed_transaction(wallet: &LocalWallet) -> Result<Transaction, E
     Ok(transaction)
 }
 
-async fn check_if_block_height_increased(
+async fn has_block_height_increased(
     provider: &Provider,
-    height_to_compare_with: u64,
+    previous_height: u64,
 ) -> Result<bool, Error> {
-    let has_block_height_increased = || async {
+    let block_height_increased = || async {
         let current_block_height = provider.latest_block_height().await?;
-        Ok(current_block_height > height_to_compare_with)
+        Ok(current_block_height > previous_height)
     };
 
-    let height_increased = retry_until(has_block_height_increased, 5, Duration::from_millis(100))
+    let height_increased = retry_until(block_height_increased, 5, Duration::from_millis(100))
         .await
         .map_err(|err| Error::NetworkError(err.to_string()))?;
 
     Ok(height_increased)
 }
 
 fn generate_no_op_script(inputs: Vec<Input>, outputs: Vec<Output>) -> Transaction {
-    let random_data = rand::thread_rng().gen::<[u8; 32]>();
-    Transaction::Script {
-        gas_price: DEFAULT_GAS_PRICE,
-        gas_limit: DEFAULT_GAS_LIMIT,
-        byte_price: DEFAULT_BYTE_PRICE,
-        maturity: DEFAULT_MATURITY,
-        receipts_root: Default::default(),
-        script: Opcode::RET(REG_ONE).to_bytes().to_vec(),
-        script_data: random_data.to_vec(),
-        inputs,
-        outputs,
-        witnesses: vec![],
-        metadata: None,
+    if let Transaction::Script {
+        gas_price,
+        gas_limit,
+        byte_price,
+        maturity,
+        receipts_root: _,
+        script,
+        script_data: _,
+        inputs: _,
+        outputs: _,
+        witnesses,
+        metadata: _,
+    } = Transaction::default()
+    {
```

Comment:
Not really convinced by this. If you are really scared that all of a sudden we break the `Transaction::default -> Transaction::Script` assertion, you can just check that the Enum variant is the right one. My idea was to reduce the number of SLOC :smile: 
I think you can remove this function altogether! See above comment.

Reply:
How would you go about checking if the variant is the right one if not by doing a `match`? And if you do a match on an exhaustive enum you need to handle all matches. And handling that requires panicking if we ever break `Transaction::default->Transaction::Script`.